### ISSUE TITLE
🐛 Bug - Deleted channel not dissapearing 

### ISSUE THREAD

**Issue by lunariixx** [Valour-Software/Valour#1096]
**Describe the bug**
Deleting a channel does not cause it to disappear until the browser tab is reloaded.

**To Reproduce**
1. Create a channel
2. Delete said channel
3. See error

**Expected behavior**
The channel to disappear 

**Screenshots**

**System Information:**
 - OS: Arch Linux
 - Valour Version V0.2.1.0
 - Valour Platform: Web
     - Browser: Firefox

**Additional context**

**Comment by darkwiz666** [comment]
Bumping issue as it is STILL occurring. Error also occurs when attempting to change channel or category name;

> PUT https://app.valour.gg/api/channels/28843742699978752 405 (Method Not Allowed)

**Comment by SpikeViper** [comment]
You seem to be stuck on a very old version. Current is V0.3.5.91 (f80dc4b). Old versions had an issue where the cache failed to be cleared, so try manually emptying your browser cache or going into the Application Data tab in devtools (f12) and deleting all data.

**Comment by darkwiz666** [comment]
No, that was the version the error was happening. I tried this again today and the version is .3.5.91(869a315) and it's the exact same issue.

Just to drive home the point: I can delete my planets, create channels and categories, modify the category description, but I can't delete them or the channels within them.
